src/vs/workbench/browser/parts/statusbar/media/statusbarpart.css

```diff
@@ -83,7 +90,7 @@
 }
 
 .monaco-workbench .part.statusbar > .items-container > .statusbar-item.right.last-visible-item {
-	padding-right: 7px; /* Add padding to the most right status bar item */
+	margin-right: 7px; /* Add padding to the most right status bar item */
```

Comment:
The centering of the beak is done now based on the width of the status item.
Since the padding is included within the width for the bell, the beak doesn't show at the center for the bell but shows correctly for other status items, unless we change it to margin-right.